datafusion/core/tests/tpcds_planning.rs

```diff
@@ -801,6 +801,7 @@ async fn tpcds_physical_q53() -> Result<()> {
     create_physical_plan(53).await
 }
 
+#[ignore]
```

Comment:
why is it ignored

Reply:
Oops, I forgot to delete that. I've been encountering a stackoverflow error whenever I run `cargo test`, and this issue is present both in the main branch and in this PR.

Reply:
removed